sys-auth/elogind/elogind-255.5-r1.ebuild

```
@@ -172,6 +172,18 @@ pkg_postinst() {
 			elog "configuration remember to migrate those to new configuration file."
 		fi
 	done
+
+	local file files
+	# find custom hooks excluding known (nvidia-drivers, sys-power/tlp)
+	readarray -t files <<<$(find /$(get_libdir)/elogind/system-sleep/ -type f \
```

<<< always adds a newline and so ${files[@]} is never empty causing an empty:
```
 * *** Custom hooks in obsolete path detected ***
 * Move these custom hooks to /etc/elogind/system-sleep/ instead.
```
```suggestion
	readarray -t files < <(find "${EROOT}"/$(get_libdir)/elogind/system-sleep/ -type f \
```